themes/inbeat/layouts/singles/collaboration-cost-calculator.html

```diff
@@ -295,6 +297,9 @@ <h1>{{- .Params.titre -}}</h1>
                   this.error = 'Please enter a username.';
                   return;
               }
+              if (!this.initialSearch) {
+                this.showUnlockPopup(this.cookieName)
```

Comment:
I don't think it takes a `string` param

Reply:
What do you mean? I believe it works with string too?

Reply:
The method `showUnlockPopup` doesn't take a parameter. It should be called without any parameters.
![CleanShot 2023-10-27 at 07 57 07@2x](https://github.com/inBeat/inbeat-website/assets/18277370/a04ec7c9-7e3f-43ea-8b37-981072ae88d0)